cli/cmd/deploy/deploy.go

```diff
@@ -107,10 +111,19 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				}
 
 				if err := auth.Login(ctx, cfg, redirectURL); err != nil {
+					if errors.Is(err, deviceauth.ErrAuthenticationTimedout) {
+						warn.Println("Rill login timed out because you did not confirm the code in the browser.")
```

Comment:
@begelundmuller Can the login prompt timeout for any other error apart from user not confirming the code ?

Reply:
change text to - "Rill login has timed out as the code was not confirmed in the browser."